docs/index.md

```diff
@@ -977,31 +975,31 @@ Mocha does not retry test failures by default.
 
 ### `--slow <ms>, -s <ms>`
 
-Specify the "slow" test threshold in milliseconds.  Mocha uses this to highlight test cases that are taking too long.  "Slow" tests are not considered failures.
+Specify the "slow" test threshold in milliseconds. Mocha uses this to highlight test cases that are taking too long. "Slow" tests are not considered failures.
 
-Note: A test that executes for *half* of the "slow" time will be highlighted *in yellow* with the default `spec` reporter; a test that executes for entire "slow" time will be highlighted *in red*.
+Note: A test that executes for _half_ of the "slow" time will be highlighted _in yellow_ with the default `spec` reporter; a test that executes for entire "slow" time will be highlighted _in red_.
 
 ### `--timeout <ms>, -t <ms>`
```

Comment:
BTW, `--timeouts` has been added as well. No idea why though since it's a single value not capable of multiple values.

Reply:
`--timeouts` is the opposite of `--no-timeouts`

Reply:
Logically, sure... don't see your point.

Question was why was `--timeouts` added at all, given that we already have `--timeout` which was handling the task just fine. And the existing matches preconceived expectation -- singular name accepting singular value.